src/ObjCRuntime/Runtime.cs

```diff
@@ -50,6 +50,7 @@ public partial class Runtime {
 
 		internal const uint INVALID_TOKEN_REF = 0xFFFFFFFF;
 
+#pragma warning disable 649 // Field 'X' is never assigned to, and will always have its default value
 		internal unsafe struct MTRegistrationMap {
 			public IntPtr assembly;
```

Comment:
Why not just set it to IntPtr.Zero?

Reply:
That might add code to any constructors / etc that we don't need.